lumigator/jobs/inference/inference.py

```diff
@@ -92,7 +92,7 @@ def run_inference(config: InferenceJobConfig) -> Path:
     elif config.hf_pipeline:
         logger.info(f"Using HuggingFace client with model {config.hf_pipeline.model_name_or_path}.")
         model_client = HuggingFaceModelClient(config)
-        output_model_name = config.hf_pipeline.model
+        output_model_name = config.hf_pipeline.model_name_or_path
```

Comment:
Good catch! Now I'm confused about how this bug made it through my testing in the litellm pr...

Reply:
Dunno T_T It may be that this parameter is actually not needed at all! Could you please chase its use in the inference job?

Reply:
Drumroll please! 🥁 

https://github.com/mozilla-ai/lumigator/blob/main/lumigator/jobs/inference/inference_config.py#L129

model is an alias for model_name_or_path 😆

Reply:
Thanks for checking it out 😄